tests/sentry/web/frontend/test_auth_mobile.py

```diff
@@ -0,0 +1,209 @@
+from urllib.parse import parse_qs, urlparse
```

Comment:
I think this test should be moved to the backend, likely in https://github.com/getsentry/sentry/tree/master/tests/sentry/auth